### ISSUE TITLE
ActivityNotFoundException With NativeBridgeActivity.java:86

### ISSUE THREAD

**Issue by tim12332000** [playgameservices/play-games-plugin-for-unity#2651]
I got this exception and crash.
```
Caused by: android.content.ActivityNotFoundException: 
  at android.app.Instrumentation.checkStartActivityResult (Instrumentation.java:1944)
  at android.app.Instrumentation.execStartActivity (Instrumentation.java:1618)
  at android.app.Activity.startActivityForResult (Activity.java:4668)
  at android.app.Activity.startActivityForResult (Activity.java:4599)
  at com.google.games.bridge.NativeBridgeActivity.startActivityForResult (NativeBridgeActivity.java:86)
  at com.google.games.bridge.NativeBridgeActivity.onStart (NativeBridgeActivity.java:62)
  at android.app.Instrumentation.callActivityOnStart (Instrumentation.java:1335)
  at android.app.Activity.performStart (Activity.java:7245)
  at android.app.ActivityThread.performLaunchActivity (ActivityThread.java:2912)
```
So I write some code to try reproduce this error:

```
PlayGamesClientConfiguration config = new PlayGamesClientConfiguration.Builder()
	   .WithInvitationDelegate(InvitationReceivedDelegate)
	   .WithMatchDelegate(MatchDelegate)
	   .Build();
PlayGamesPlatform.InitializeInstance(config);
PlayGamesPlatform.Activate();
GooglePlayGames.PlayGamesPlatform.Instance.Authenticate(AuthenticateRes, false);
```
And I trace code form here:
AndroidClient.cs line47
`config.SetOptionalIntentHandlerForUI((intent) =>`

AndroidPlatformConfiguration.cs line 37.
`internal void SetOptionalIntentHandlerForUI(Action<IntPtr> intentHandler)`

It did not callback  , So I confuse what is it , and how to work?
How can I reproduce this error? 

thanks.

**Comment by legas1** [comment]
Does this error occure even with Force Resolve option inResolver?

**Comment by tim12332000** [comment]
@legas1  "Force Resolve option in Resolver" What it mean ?

**Comment by tim12332000** [comment]
I know how to use Trigger `config.SetOptionalIntentHandlerForUI((intent) =>` callback , 
just invoke `PlayGamesPlatform.Instance.ShowLeaderboardUI();  `

**Comment by Watcher3056** [comment]
Got also some similar
`
CPH1805, Android 8.1

java.lang.Error: FATAL EXCEPTION [main]
Unity version     : 2018.4.4f1
Device model      : OPPO CPH1805
Device fingerprint: OPPO/CPH1805/CPH1805:8.1.0/OPM1.171019.026/1555507955:user/release-keys

Caused by
 
  at android.app.ActivityThread.performLaunchActivity (ActivityThread.java:2978)
 
  at android.app.ActivityThread.handleLaunchActivity (ActivityThread.java:3056)
 
  at android.app.ActivityThread.-wrap11 (Unknown Source)
 
  at android.app.ActivityThread$H.handleMessage (ActivityThread.java:1774)
 
  at android.os.Handler.dispatchMessage (Handler.java:106)
 
  at android.os.Looper.loop (Looper.java:198)
 
  at android.app.ActivityThread.main (ActivityThread.java:7038)
 
  at java.lang.reflect.Method.invoke (Method.java)
 
  at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run (RuntimeInit.java:523)
 
  at com.android.internal.os.ZygoteInit.main (ZygoteInit.java:836)
Caused by: android.content.ActivityNotFoundException: 	at android.app.Instrumentation.checkStartActivityResult(Instrumentation.java:1944)
	at android.app.Instrumentation.execStartActivity(Instrumentation.java:1618)
	at android.app.Activity.startActivityForResult(Activity.java:4711)
	at android.app.Activity.startActivityForResult(Activity.java:4641)
	at com.google.games.bridge.NativeBridgeActivity.startActivityForResult(NativeBridgeActivity.java:86)
	at com.google.games.bridge.NativeBridgeActivity.onStart(NativeBridgeActivity.java:62)
	at android.app.Instrumentation.callActivityOnStart(Instrumentation.java:1335)
	at android.app.Activity.performStart(Activity.java:7341)
	at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:2941)
	... 9 more
`

**Comment by tim12332000** [comment]
@Watcher3056  , my crash log is same on "OPPO CPH1853" !? , Is possible with specific device ?
```
java.lang.Error: FATAL EXCEPTION [main]
Unity version     : 2018.4.2f1
Device model      : OPPO CPH1853
Device fingerprint: OPPO/CPH1853/CPH1853:8.1.0/OPM1.171019.026/1559663459:user/release-keys
```

**Comment by svennnni** [comment]
Also getting these on some devices after a recent update:

No Activity found to handle Intent { act=com.google.android.gms.games.VIEW_LEADERBOARDS dat=version:12451000 flg=0x4000000 pkg=com.google.android.play.games (has extras) }
	
Unity version     : 2018.4.5f1
Device model      : Alcatel 5041C
Device fingerprint: TCL/5041C/U50A_ATT:8.1.0/OPM1.171019.011/v8MAB-0:user/release-keys

Unity version     : 2018.4.5f1
Device model      : LGE LG-K520
Device fingerprint: lge/ph1n_global_com/ph1n:6.0.1/MMB29M/1813507546d85:user/release-keys

**Comment by olehkuznetsov** [comment]
Should be fixed in v0.10.06
